README.md

```diff
@@ -7,6 +7,7 @@
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/f3732b3327e34358a0e9d1fe9f661f08)](https://www.codacy.com/app/nlohmann/json?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=nlohmann/json&amp;utm_campaign=Badge_Grade)
 [![Try online](https://img.shields.io/badge/try-online-blue.svg)](https://wandbox.org/permlink/yCTCvQi0ZVxGuL3L)
 [![Documentation](https://img.shields.io/badge/docs-doxygen-blue.svg)](http://nlohmann.github.io/json)
+[![Hunter package](https://img.shields.io/badge/hunter-nlohmann_json-blue.svg)](https://docs.hunter.sh/en/latest/packages/pkg/nlohmann_json.html)
```

Comment:
Let's not add a badge for Hunter. This is too prominent, and there are many other package managers.